app/src/ui/branches/ci-status.tsx

```diff
@@ -35,10 +35,15 @@ export class CIStatus extends React.Component<ICIStatusProps, {}> {
 }
 
 function generateStatusHistory(prStatus: PullRequestStatus): string {
-  return prStatus.statuses.reduce(
-    (prev, curr) => `${prev}\n${curr.description}`,
-    `Commit status: ${prStatus.state}`
-  )
+  const statusCount = prStatus.statuses.length || 0
+
+  if (statusCount === 0) {
+    return prStatus.state.toUpperCase()
+  } else {
+    const successCount = prStatus.statuses.filter(x => x.state === 'success')
```

Comment:
Maybe `successes` here? `successCount.length` doesn’t make sense here. (or you could put the `.length` part on this line)

Reply:
This isn't ready for review.

Reply:
Sorry, didn’t check for the label before commenting.

Reply:
Now it is :)

Reply:
Fixed. Just now 👍